lib/VisualBuilder.js

```diff
@@ -312,22 +312,28 @@ class VisualBuilder extends EventEmitter {
     }
 
     _getLocalization(visualConfig) {
+        let stringResourcesPath = this.visualPackage.buildPath('.api', 'v' + visualConfig.apiVersion, 'schema.stringResources.json');
+
+        if (!fs.existsSync(stringResourcesPath)) {
+            return undefined;
+        }
+
         let localeResourcesFiles = visualConfig.stringResources ? visualConfig.stringResources.map((path) => this.visualPackage.buildPath(path)) : [];
         return this._getAvialableLocalePaths()
             .then((localeDirPaths) => {
                 if (!localeDirPaths || !localeDirPaths.length) {
                     if (!localeResourcesFiles) { return; }
                     // add only string resources localization 
                     return Promise.all([
-                        this._getValidationSchema(this.visualPackage.buildPath('.api', 'v' + visualConfig.apiVersion, 'schema.stringResources.json')),
+                        this._getValidationSchema(stringResourcesPath),
                         this._getLocaleFromStringResources(localeResourcesFiles)
                     ])
                         .then(([schema, localeStringJsons]) =>
                             Promise.all(localeStringJsons.map(localeJson => this._validateJsonBySchema(localeJson, schema))));
                 }
 
                 return Promise.all([
-                    this._getValidationSchema(this.visualPackage.buildPath('.api', 'v' + visualConfig.apiVersion, 'schema.stringResources.json')),
+                    this._getValidationSchema(stringResourcesPath),
```

Comment:
Looks nice but we must include a unit test that covers this change and confirms that the code works properly

Reply:
Ok. I will add